examples/lstv-based-vad/ARMA.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import division
+import numpy as np
+from scipy.io import wavfile
+from scipy import signal
+import matplotlib.pyplot as plt
+import matplotlib.mlab as mlb
+
+
+def ApplyARMA(in_matrix, order):
+  """This function applies Autoregressive Moving Averarge(ARMA)
+     on the rows of the input matrix.
+     
+  Args:
+      in_matrix: the input matrix
+      order: the order of the ARMA process
+
+  Output:
```

Comment:
Use "Returns" instead of "Output" in docstrings. This comment applies to a number of files in this PR.

[Sphinx](https://www.sphinx-doc.org/en/master/index.html) has special handling for "Results" sections in [Google style docstrings](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) which is what we are using. If we move this file out of examples -- I think ARMA is generally useful to have --, Sphinx would generate a nice documentation page for it.